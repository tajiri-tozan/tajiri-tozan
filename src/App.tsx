import React from "react";
import { BrowserRouter } from "react-router-dom";
import { ReactGenieAnimations } from "react-genie-styled-components";
import "./App.css";
import RootRouter from "pages";

function App() {
  return (
    <BrowserRouter>
      <ReactGenieAnimations />
      <RootRouter />
    </BrowserRouter>
  );
}

export default App;
