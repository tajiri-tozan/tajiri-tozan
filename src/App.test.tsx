import React from 'react';
import { render, waitFor } from '@testing-library/react';
import App from './App';

it('renders Tajiri Tozan', async () => {
  const { getByText } = render(<App />);

  await waitFor(() => {
    getByText(/Tajiri Tozan/i);
  });
});

it('snapshot testing', () => {
  const { asFragment } = render(<App />);
  expect(asFragment()).toMatchSnapshot();
});
