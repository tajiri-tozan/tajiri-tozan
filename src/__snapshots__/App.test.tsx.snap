// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot testing 1`] = `
<DocumentFragment>
  <div
    class="Top__Contents-sc-11zlplz-0 eWGYWI"
  >
    <header
      class="Top__Header-sc-11zlplz-1 TWVhC"
    >
      <h1
        class="Top__SiteTitle-sc-11zlplz-2 ejlovn"
      >
        Tajiri Tozan
      </h1>
    </header>
    <div
      class="makeStyles-root-19"
    >
      <ul
        class="MuiGridList-root makeStyles-gridList-20"
        style="margin: -50px;"
      >
        <li
          class="MuiGridListTile-root"
          style="width: 100%; height: auto; padding: 50px;"
        >
          <div
            class="MuiGridListTile-tile"
          >
            <img
              alt="陣馬山"
              class="Top__YamaImg-sc-11zlplz-3 eoZknr"
              src="/images/20141207-jinbasan.jpg"
            />
            <div
              class="MuiGridListTileBar-root MuiGridListTileBar-titlePositionBottom MuiGridListTileBar-rootSubtitle"
            >
              <div
                class="MuiGridListTileBar-titleWrap"
              >
                <div
                  class="MuiGridListTileBar-title"
                >
                  陣馬山
                </div>
                <div
                  class="MuiGridListTileBar-subtitle"
                >
                  <span>
                    2014/12/07
                  </span>
                </div>
              </div>
            </div>
          </div>
        </li>
        <li
          class="MuiGridListTile-root"
          style="width: 100%; height: auto; padding: 50px;"
        >
          <div
            class="MuiGridListTile-tile"
          >
            <img
              alt="外秩父七峰縦走"
              class="Top__YamaImg-sc-11zlplz-3 eoZknr"
              src="/images/20150328-chichibu.jpg"
            />
            <div
              class="MuiGridListTileBar-root MuiGridListTileBar-titlePositionBottom MuiGridListTileBar-rootSubtitle"
            >
              <div
                class="MuiGridListTileBar-titleWrap"
              >
                <div
                  class="MuiGridListTileBar-title"
                >
                  外秩父七峰縦走
                </div>
                <div
                  class="MuiGridListTileBar-subtitle"
                >
                  <span>
                    2015/03/28
                  </span>
                </div>
              </div>
            </div>
          </div>
        </li>
        <li
          class="MuiGridListTile-root"
          style="width: 100%; height: auto; padding: 50px;"
        >
          <div
            class="MuiGridListTile-tile"
          >
            <img
              alt="奥多摩"
              class="Top__YamaImg-sc-11zlplz-3 eoZknr"
              src="/images/20150501-okutama.jpg"
            />
            <div
              class="MuiGridListTileBar-root MuiGridListTileBar-titlePositionBottom MuiGridListTileBar-rootSubtitle"
            >
              <div
                class="MuiGridListTileBar-titleWrap"
              >
                <div
                  class="MuiGridListTileBar-title"
                >
                  奥多摩
                </div>
                <div
                  class="MuiGridListTileBar-subtitle"
                >
                  <span>
                    2015/05/01
                  </span>
                </div>
              </div>
            </div>
          </div>
        </li>
        <li
          class="MuiGridListTile-root"
          style="width: 100%; height: auto; padding: 50px;"
        >
          <div
            class="MuiGridListTile-tile"
          >
            <img
              alt="塩山"
              class="Top__YamaImg-sc-11zlplz-3 eoZknr"
              src="/images/20150501-enzan.jpg"
            />
            <div
              class="MuiGridListTileBar-root MuiGridListTileBar-titlePositionBottom MuiGridListTileBar-rootSubtitle"
            >
              <div
                class="MuiGridListTileBar-titleWrap"
              >
                <div
                  class="MuiGridListTileBar-title"
                >
                  塩山
                </div>
                <div
                  class="MuiGridListTileBar-subtitle"
                >
                  <span>
                    2015/06/14
                  </span>
                </div>
              </div>
            </div>
          </div>
        </li>
        <li
          class="MuiGridListTile-root"
          style="width: 100%; height: auto; padding: 50px;"
        >
          <div
            class="MuiGridListTile-tile"
          >
            <img
              alt="尾瀬"
              class="Top__YamaImg-sc-11zlplz-3 eoZknr"
              src="/images/20150704-oze.jpg"
            />
            <div
              class="MuiGridListTileBar-root MuiGridListTileBar-titlePositionBottom MuiGridListTileBar-rootSubtitle"
            >
              <div
                class="MuiGridListTileBar-titleWrap"
              >
                <div
                  class="MuiGridListTileBar-title"
                >
                  尾瀬
                </div>
                <div
                  class="MuiGridListTileBar-subtitle"
                >
                  <span>
                    2015/07/04
                  </span>
                </div>
              </div>
            </div>
          </div>
        </li>
        <li
          class="MuiGridListTile-root"
          style="width: 100%; height: auto; padding: 50px;"
        >
          <div
            class="MuiGridListTile-tile"
          >
            <img
              alt="槍ヶ岳"
              class="Top__YamaImg-sc-11zlplz-3 eoZknr"
              src="/images/20150920-yarigadake.jpg"
            />
            <div
              class="MuiGridListTileBar-root MuiGridListTileBar-titlePositionBottom MuiGridListTileBar-rootSubtitle"
            >
              <div
                class="MuiGridListTileBar-titleWrap"
              >
                <div
                  class="MuiGridListTileBar-title"
                >
                  槍ヶ岳
                </div>
                <div
                  class="MuiGridListTileBar-subtitle"
                >
                  <span>
                    2015/09/20
                  </span>
                </div>
              </div>
            </div>
          </div>
        </li>
        <li
          class="MuiGridListTile-root"
          style="width: 100%; height: auto; padding: 50px;"
        >
          <div
            class="MuiGridListTile-tile"
          >
            <img
              alt="三つ峠山"
              class="Top__YamaImg-sc-11zlplz-3 eoZknr"
              src="/images/20160423-mitsutoge.jpg"
            />
            <div
              class="MuiGridListTileBar-root MuiGridListTileBar-titlePositionBottom MuiGridListTileBar-rootSubtitle"
            >
              <div
                class="MuiGridListTileBar-titleWrap"
              >
                <div
                  class="MuiGridListTileBar-title"
                >
                  三つ峠山
                </div>
                <div
                  class="MuiGridListTileBar-subtitle"
                >
                  <span>
                    2016/04/23
                  </span>
                </div>
              </div>
            </div>
          </div>
        </li>
        <li
          class="MuiGridListTile-root"
          style="width: 100%; height: auto; padding: 50px;"
        >
          <div
            class="MuiGridListTile-tile"
          >
            <img
              alt="大菩薩嶺"
              class="Top__YamaImg-sc-11zlplz-3 eoZknr"
              src="/images/20160502-daibosatsurei.jpg"
            />
            <div
              class="MuiGridListTileBar-root MuiGridListTileBar-titlePositionBottom MuiGridListTileBar-rootSubtitle"
            >
              <div
                class="MuiGridListTileBar-titleWrap"
              >
                <div
                  class="MuiGridListTileBar-title"
                >
                  大菩薩嶺
                </div>
                <div
                  class="MuiGridListTileBar-subtitle"
                >
                  <span>
                    2016/05/02
                  </span>
                </div>
              </div>
            </div>
          </div>
        </li>
        <li
          class="MuiGridListTile-root"
          style="width: 100%; height: auto; padding: 50px;"
        >
          <div
            class="MuiGridListTile-tile"
          >
            <img
              alt="燕岳"
              class="Top__YamaImg-sc-11zlplz-3 eoZknr"
              src="/images/20160904-tsubakuro.jpg"
            />
            <div
              class="MuiGridListTileBar-root MuiGridListTileBar-titlePositionBottom MuiGridListTileBar-rootSubtitle"
            >
              <div
                class="MuiGridListTileBar-titleWrap"
              >
                <div
                  class="MuiGridListTileBar-title"
                >
                  燕岳
                </div>
                <div
                  class="MuiGridListTileBar-subtitle"
                >
                  <span>
                    2016/09/04
                  </span>
                </div>
              </div>
            </div>
          </div>
        </li>
        <li
          class="MuiGridListTile-root"
          style="width: 100%; height: auto; padding: 50px;"
        >
          <div
            class="MuiGridListTile-tile"
          >
            <img
              alt="鍋割山"
              class="Top__YamaImg-sc-11zlplz-3 eoZknr"
              src="/images/20161229-tsubakuro.jpg"
            />
            <div
              class="MuiGridListTileBar-root MuiGridListTileBar-titlePositionBottom MuiGridListTileBar-rootSubtitle"
            >
              <div
                class="MuiGridListTileBar-titleWrap"
              >
                <div
                  class="MuiGridListTileBar-title"
                >
                  鍋割山
                </div>
                <div
                  class="MuiGridListTileBar-subtitle"
                >
                  <span>
                    2016/12/29
                  </span>
                </div>
              </div>
            </div>
          </div>
        </li>
        <li
          class="MuiGridListTile-root"
          style="width: 100%; height: auto; padding: 50px;"
        >
          <div
            class="MuiGridListTile-tile"
          >
            <img
              alt="川乗山"
              class="Top__YamaImg-sc-11zlplz-3 eoZknr"
              src="/images/20170502-kawanoriyama.jpg"
            />
            <div
              class="MuiGridListTileBar-root MuiGridListTileBar-titlePositionBottom MuiGridListTileBar-rootSubtitle"
            >
              <div
                class="MuiGridListTileBar-titleWrap"
              >
                <div
                  class="MuiGridListTileBar-title"
                >
                  川乗山
                </div>
                <div
                  class="MuiGridListTileBar-subtitle"
                >
                  <span>
                    2017/05/02
                  </span>
                </div>
              </div>
            </div>
          </div>
        </li>
        <li
          class="MuiGridListTile-root"
          style="width: 100%; height: auto; padding: 50px;"
        >
          <div
            class="MuiGridListTile-tile"
          >
            <img
              alt="塔ノ岳"
              class="Top__YamaImg-sc-11zlplz-3 eoZknr"
              src="/images/20170603-tonodake.jpg"
            />
            <div
              class="MuiGridListTileBar-root MuiGridListTileBar-titlePositionBottom MuiGridListTileBar-rootSubtitle"
            >
              <div
                class="MuiGridListTileBar-titleWrap"
              >
                <div
                  class="MuiGridListTileBar-title"
                >
                  塔ノ岳
                </div>
                <div
                  class="MuiGridListTileBar-subtitle"
                >
                  <span>
                    2017/06/03
                  </span>
                </div>
              </div>
            </div>
          </div>
        </li>
        <li
          class="MuiGridListTile-root"
          style="width: 100%; height: auto; padding: 50px;"
        >
          <div
            class="MuiGridListTile-tile"
          >
            <img
              alt="谷川岳"
              class="Top__YamaImg-sc-11zlplz-3 eoZknr"
              src="/images/20170722-tanigawadake.jpg"
            />
            <div
              class="MuiGridListTileBar-root MuiGridListTileBar-titlePositionBottom MuiGridListTileBar-rootSubtitle"
            >
              <div
                class="MuiGridListTileBar-titleWrap"
              >
                <div
                  class="MuiGridListTileBar-title"
                >
                  谷川岳
                </div>
                <div
                  class="MuiGridListTileBar-subtitle"
                >
                  <span>
                    2017/07/22
                  </span>
                </div>
              </div>
            </div>
          </div>
        </li>
        <li
          class="MuiGridListTile-root"
          style="width: 100%; height: auto; padding: 50px;"
        >
          <div
            class="MuiGridListTile-tile"
          >
            <img
              alt="富士山"
              class="Top__YamaImg-sc-11zlplz-3 eoZknr"
              src="/images/20170819-fuji.jpg"
            />
            <div
              class="MuiGridListTileBar-root MuiGridListTileBar-titlePositionBottom MuiGridListTileBar-rootSubtitle"
            >
              <div
                class="MuiGridListTileBar-titleWrap"
              >
                <div
                  class="MuiGridListTileBar-title"
                >
                  富士山
                </div>
                <div
                  class="MuiGridListTileBar-subtitle"
                >
                  <span>
                    2017/08/19
                  </span>
                </div>
              </div>
            </div>
          </div>
        </li>
        <li
          class="MuiGridListTile-root"
          style="width: 100%; height: auto; padding: 50px;"
        >
          <div
            class="MuiGridListTile-tile"
          >
            <img
              alt="北岳"
              class="Top__YamaImg-sc-11zlplz-3 eoZknr"
              src="/images/20170930-kitadake.jpg"
            />
            <div
              class="MuiGridListTileBar-root MuiGridListTileBar-titlePositionBottom MuiGridListTileBar-rootSubtitle"
            >
              <div
                class="MuiGridListTileBar-titleWrap"
              >
                <div
                  class="MuiGridListTileBar-title"
                >
                  北岳
                </div>
                <div
                  class="MuiGridListTileBar-subtitle"
                >
                  <span>
                    2017/09/30
                  </span>
                </div>
              </div>
            </div>
          </div>
        </li>
        <li
          class="MuiGridListTile-root"
          style="width: 100%; height: auto; padding: 50px;"
        >
          <div
            class="MuiGridListTile-tile"
          >
            <img
              alt="金時山"
              class="Top__YamaImg-sc-11zlplz-3 eoZknr"
              src="/images/20171229-kintokisan.jpg"
            />
            <div
              class="MuiGridListTileBar-root MuiGridListTileBar-titlePositionBottom MuiGridListTileBar-rootSubtitle"
            >
              <div
                class="MuiGridListTileBar-titleWrap"
              >
                <div
                  class="MuiGridListTileBar-title"
                >
                  金時山
                </div>
                <div
                  class="MuiGridListTileBar-subtitle"
                >
                  <span>
                    2017/12/29
                  </span>
                </div>
              </div>
            </div>
          </div>
        </li>
        <li
          class="MuiGridListTile-root"
          style="width: 100%; height: auto; padding: 50px;"
        >
          <div
            class="MuiGridListTile-tile"
          >
            <img
              alt="瑞牆山"
              class="Top__YamaImg-sc-11zlplz-3 eoZknr"
              src="/images/20180421-mizugakiyama.jpg"
            />
            <div
              class="MuiGridListTileBar-root MuiGridListTileBar-titlePositionBottom MuiGridListTileBar-rootSubtitle"
            >
              <div
                class="MuiGridListTileBar-titleWrap"
              >
                <div
                  class="MuiGridListTileBar-title"
                >
                  瑞牆山
                </div>
                <div
                  class="MuiGridListTileBar-subtitle"
                >
                  <span>
                    2018/04/21
                  </span>
                </div>
              </div>
            </div>
          </div>
        </li>
        <li
          class="MuiGridListTile-root"
          style="width: 100%; height: auto; padding: 50px;"
        >
          <div
            class="MuiGridListTile-tile"
          >
            <img
              alt="八ヶ岳"
              class="Top__YamaImg-sc-11zlplz-3 eoZknr"
              src="/images/20180721-yatsugatake.jpg"
            />
            <div
              class="MuiGridListTileBar-root MuiGridListTileBar-titlePositionBottom MuiGridListTileBar-rootSubtitle"
            >
              <div
                class="MuiGridListTileBar-titleWrap"
              >
                <div
                  class="MuiGridListTileBar-title"
                >
                  八ヶ岳
                </div>
                <div
                  class="MuiGridListTileBar-subtitle"
                >
                  <span>
                    2018/07/21
                  </span>
                </div>
              </div>
            </div>
          </div>
        </li>
        <li
          class="MuiGridListTile-root"
          style="width: 100%; height: auto; padding: 50px;"
        >
          <div
            class="MuiGridListTile-tile"
          >
            <img
              alt="天城山"
              class="Top__YamaImg-sc-11zlplz-3 eoZknr"
              src="/images/20190217-amagisan.jpg"
            />
            <div
              class="MuiGridListTileBar-root MuiGridListTileBar-titlePositionBottom MuiGridListTileBar-rootSubtitle"
            >
              <div
                class="MuiGridListTileBar-titleWrap"
              >
                <div
                  class="MuiGridListTileBar-title"
                >
                  天城山
                </div>
                <div
                  class="MuiGridListTileBar-subtitle"
                >
                  <span>
                    2019/02/17
                  </span>
                </div>
              </div>
            </div>
          </div>
        </li>
        <li
          class="MuiGridListTile-root"
          style="width: 100%; height: auto; padding: 50px;"
        >
          <div
            class="MuiGridListTile-tile"
          >
            <img
              alt="武甲山"
              class="Top__YamaImg-sc-11zlplz-3 eoZknr"
              src="/images/20190330-bukousan.jpg"
            />
            <div
              class="MuiGridListTileBar-root MuiGridListTileBar-titlePositionBottom MuiGridListTileBar-rootSubtitle"
            >
              <div
                class="MuiGridListTileBar-titleWrap"
              >
                <div
                  class="MuiGridListTileBar-title"
                >
                  武甲山
                </div>
                <div
                  class="MuiGridListTileBar-subtitle"
                >
                  <span>
                    2019/03/30
                  </span>
                </div>
              </div>
            </div>
          </div>
        </li>
        <li
          class="MuiGridListTile-root"
          style="width: 100%; height: auto; padding: 50px;"
        >
          <div
            class="MuiGridListTile-tile"
          >
            <img
              alt="乾徳山"
              class="Top__YamaImg-sc-11zlplz-3 eoZknr"
              src="/images/20190721-kentokusan.jpg"
            />
            <div
              class="MuiGridListTileBar-root MuiGridListTileBar-titlePositionBottom MuiGridListTileBar-rootSubtitle"
            >
              <div
                class="MuiGridListTileBar-titleWrap"
              >
                <div
                  class="MuiGridListTileBar-title"
                >
                  乾徳山
                </div>
                <div
                  class="MuiGridListTileBar-subtitle"
                >
                  <span>
                    2019/07/21
                  </span>
                </div>
              </div>
            </div>
          </div>
        </li>
        <li
          class="MuiGridListTile-root"
          style="width: 100%; height: auto; padding: 50px;"
        >
          <div
            class="MuiGridListTile-tile"
          >
            <img
              alt="金時山"
              class="Top__YamaImg-sc-11zlplz-3 eoZknr"
              src="/images/20191214-kintokisan.jpg"
            />
            <div
              class="MuiGridListTileBar-root MuiGridListTileBar-titlePositionBottom MuiGridListTileBar-rootSubtitle"
            >
              <div
                class="MuiGridListTileBar-titleWrap"
              >
                <div
                  class="MuiGridListTileBar-title"
                >
                  金時山
                </div>
                <div
                  class="MuiGridListTileBar-subtitle"
                >
                  <span>
                    2019/12/14
                  </span>
                </div>
              </div>
            </div>
          </div>
        </li>
      </ul>
    </div>
  </div>
</DocumentFragment>
`;
